license-information-windows.md

```diff
@@ -6,26 +6,13 @@ apply to your use. By using any or all of these files you agree to their associa
 
 This document is provided for informative purposes only, and is not itself a license.
 
-The following binaries are licensed with the
-[.NET Library License](https://dotnet.microsoft.com/dotnet_library_license.htm)
-
-* Microsoft.DiaSymReader.Native.{x86|amd64|arm|arm64}.dll (used by .NET runtime and SDK)
-* PresentationNative_cor3.dll (used by WPF)
-* vcruntime140_cor3.dll (used by WPF)
-* wpfgfx_cor3.dll (used by WPF)
-
 The following binaries are licensed with the
 [Windows SDK License](https://learn.microsoft.com/legal/windows-sdk/license):
 
 * D3DCompiler_47_cor3.dll (used by WPF)
 
-.NET 6 and earlier:
-
-* api-ms-\*.\* (used by .NET runtime)
-* ucrtbase.dll (used by .NET runtime)
-
 All other binaries and files are licensed with the
-[MIT license](https://github.com/dotnet/core/blob/main/LICENSE.TXT).
+[.NET Library License](https://github.com/dotnet/core/blob/main/LICENSE.TXT).
```

Comment:
The link still points to MIT

Reply:
Oops. Not sure how that happened. I had intended to make that change. Will resolve.